compiler/rustc_typeck/src/impl_wf_check/min_specialization.rs

```diff
@@ -384,9 +389,17 @@ fn check_specialization_on<'tcx>(tcx: TyCtxt<'tcx>, predicate: ty::Predicate<'tc
                     .emit();
```

Comment:
i can't leave a comment on line (new) 385 (old 380), but we should probably just print the whole trait ref instead of just the def path...?